File: output/tbAlumno.php
<?php
class tbAlumno{
	private  $idAlumno;
	private  $nombre;
	private  $primerApellido;
	private  $segundoApellido;
	private  $sexo;
	private  $noControl;
	private  $Column;
	private  $idCarrera;

	public function getIdAlumno(){
		return $this->idAlumno;
	}
	public  function setIdAlumno($idAlumno){
		$this->idAlumno=$idAlumno;
	}
	public function getNombre(){
		return $this->nombre;
	}
	public  function setNombre($nombre){
		$this->nombre=$nombre;
	}
	public function getPrimerApellido(){
		return $this->primerApellido;
	}
	public  function setPrimerApellido($primerApellido){
		$this->primerApellido=$primerApellido;
	}
	public function getSegundoApellido(){
		return $this->segundoApellido;
	}
	public  function setSegundoApellido($segundoApellido){
		$this->segundoApellido=$segundoApellido;
	}
	public function getSexo(){
		return $this->sexo;
	}
	public  function setSexo($sexo){
		$this->sexo=$sexo;
	}
	public function getNoControl(){
		return $this->noControl;
	}
	public  function setNoControl($noControl){
		$this->noControl=$noControl;
	}
	public function getColumn(){
		return $this->Column;
	}
	public  function setColumn($Column){
		$this->Column=$Column;
	}
	public function getIdCarrera(){
		return $this->idCarrera;
	}
	public  function setIdCarrera($idCarrera){
		$this->idCarrera=$idCarrera;
	}
}
?>

File: output/tbAlumno.java
public class tbAlumno{
	private int idAlumno;
	private String nombre;
	private String primerApellido;
	private String segundoApellido;
	private String sexo;
	private String noControl;
	private int Column;
	private int idCarrera;

	public int getIdAlumno(){
		return this.idAlumno;
	}
	public void setIdAlumno(int idAlumno){
		this.idAlumno=idAlumno;
	}
	public String getNombre(){
		return this.nombre;
	}
	public void setNombre(String nombre){
		this.nombre=nombre;
	}
	public String getPrimerApellido(){
		return this.primerApellido;
	}
	public void setPrimerApellido(String primerApellido){
		this.primerApellido=primerApellido;
	}
	public String getSegundoApellido(){
		return this.segundoApellido;
	}
	public void setSegundoApellido(String segundoApellido){
		this.segundoApellido=segundoApellido;
	}
	public String getSexo(){
		return this.sexo;
	}
	public void setSexo(String sexo){
		this.sexo=sexo;
	}
	public String getNoControl(){
		return this.noControl;
	}
	public void setNoControl(String noControl){
		this.noControl=noControl;
	}
	public int getColumn(){
		return this.Column;
	}
	public void setColumn(int Column){
		this.Column=Column;
	}
	public int getIdCarrera(){
		return this.idCarrera;
	}
	public void setIdCarrera(int idCarrera){
		this.idCarrera=idCarrera;
	}
}